packages/web/components/Player/Section/PlayerHero.tsx

```diff
@@ -1,95 +1,124 @@
+
+  const loadTypes = async () => {
+    const { types: list } = await getPersonalityInfo();
+    setTypes(list);
+  };
+  useEffect(() => { loadTypes(); }, []);
 
   return (
     <ProfileSection>
       <VStack spacing={8}>
         <Avatar
-          w={{ base: '32', md: '56' }}
-          h={{ base: '32', md: '56' }}
+          w={{ base: 32, md: 56 }}
+          h={{ base: 32, md: 56 }}
           src={getPlayerImage(player)}
           name={getPlayerName(player)}
         />
         <Box textAlign="center">
-          <Text fontSize="xl" fontFamily="heading" mb="1">
+          <Text fontSize="xl" fontFamily="heading" mb={1}>
             {getPlayerName(player)}
           </Text>
           <PlayerBrightId player={player} />
         </Box>
         <Box>
           <Text>
-            {`${getPlayerDescription(player).substring(
-              0,
-              show ? getPlayerDescription(player).length : BIO_LENGTH,
-            )}${show ? '' : '...'} `}
-            {getPlayerDescription(player).length > BIO_LENGTH && (
+            {show
+              ? description
+              : `${description.substring(0, MAX_BIO_LENGTH - 9)}…`
+            }
+            {description.length > MAX_BIO_LENGTH && (
               <Text
                 as="span"
                 fontFamily="body"
                 fontSize="xs"
                 color="cyanText"
                 cursor="pointer"
-                onClick={() => setShow(!show)}
+                onClick={() => setShow(s => !s)}
+                pl={1}
               >
                 Read {show ? 'less' : 'more'}
               </Text>
             )}
           </Text>
         </Box>
 
-        <HStack mt="2">
+        <HStack mt={2}>
           <PlayerContacts player={player} />
         </HStack>
         <Box w="100%">
           <PlayerCollab player={player} />
         </Box>
-        {player.EnneagramType && (
-          <HStack spacing={4}>
-            <Image
-              w="100%"
-              maxW="4rem"
-              src={PersonalityTypes[player.EnneagramType.name].image}
-              alt={player.EnneagramType.name}
-              style={{ mixBlendMode: 'color-dodge' }}
-            />
-            <FlexContainer align="stretch">
-              <Text color="white" fontWeight="bold">
-                {player.EnneagramType.name}
-              </Text>
-              <Text color="blueLight">{player.EnneagramType.description}</Text>
-            </FlexContainer>
-          </HStack>
+        {type && types && (
+          <Flex direction="column" id="color" mb={0} w="100%">
+            <Text
+              fontSize="xs" color="blueLight"
+              casing="uppercase"
+              mb={3}
+              textAlign="left"
+            >
+              Color Disposition
+            </Text>
+            <Link
+              isExternal
+              href={`//dysbulic.github.io/5-color-radar/#/combos/${type.mask.toString(2)}`}
+              maxH='6rem'
+            >
```

Comment:
External to this PR; but a bit more context would be useful in the linked page. I would expect to at least see the colors + one-word description (eg. "savagery") there.

Reply:
Also it seems we should link to some internal page (in the wiki / notion) that describes what the colors / disposition means. Maybe with a little question mark icon or something.

Reply:
I added the MetaGame names to the [Magic quiz site](https://dysbulic.github.io/5-color-radar/#/combos/1011). That site has a link to [the definitive Magic article](https://humanparts.medium.com/the-mtg-color-wheel-c9700a7cf36d) and the [Magic wiki](https://mtg.fandom.com) for each color combination as well as some of the icons.

The color bar in the player's profile opens a link to the quiz site's [result](https://dysbulic.github.io/5-color-radar/#/combos/11011) for that combination. My idea was that they would be seeing the colors through the tests on the editing page and via a link on the display page.

The only visual indication that the bar is a link is the name gets underlined when hovered.

The four color combinations don't have a description on the wiki or in the article. I was planning on pushing the issue to the Writers once the layouts are finalized and see if there's anyone there who knows Magic well enough to write summaries for those.